### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by ilya-lavrenov**

### CODE REVIEW

**Review by azhogov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+Here is the CMake command for you:
+
+```sh
+$ cd <openvino_build_directory>
+$ cmake -DIE_EXTRA_MODULES=<openvino_contrib>/modules <openvino_source_directory>
+$ cmake --build . -j8
+```
+
+As the result, OpenVINO will be built in the <openvino_build_directory> with all modules from `openvino_contrib` repository. If you don't want all of the modules, use CMake's `BUILD_<module_name>` boolean options. Like in this example:
```
Comment (dkurt) [inline README.md:18]: What if "If you don't want all of the modules" -> "To disable specific modules"?

File: modules/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# An overview of the openvino_contrib modules
+
+This list gives an overview of all modules available inside the contrib repository. To turn off building one of these module repositories, set the names in bold below to
+
+```sh
+$ cmake -DIE_EXTRA_MODULES=<openvino_contrib>/modules -DBUILD_<module name>=OFF <openvino_source_directory>
+```
+
+* **java_api**: Inference Engine Java API -- provides Java wrappers for Inference Engine public API.
```
Comment (dkurt) [inline modules/README.md:9]: I recommend to use a link `[java_api](./java_api)`

File: README.md (hunk 1 above)
Comment (ilya-lavrenov) [reply README.md]: Agree, fixed

File: modules/README.md (hunk 2 above)
Comment (ilya-lavrenov) [reply modules/README.md]: added a link

**Pull request merged**